### DISCUSSION TITLE
Wi-Fi Connection and Height Mapping for PCB Milling in bCNC

### DISCUSSION THREAD

**Discussion by Jairolaya12x** [vlachoudis/bCNC#1909] [Q&A] [answered]
Hello everyone,

I’m new to the CNC world and still learning. I currently have a CNC machine that needs to be connected directly to a PC, but I’m thinking of upgrading to a board with Wi-Fi capabilities.

My main goal is to mill PCBs, and I’d like to know if it’s possible to use bCNC over Wi-Fi and still perform height mapping (autoleveling) for accurate PCB milling.

Any advice on the setup or recommendations would be greatly appreciated!

Thanks in advance!

**Comment by Harvie** [comment]
ESP32 based boards?

https://github.com/bdring/Grbl_Esp32
https://github.com/grblHAL

**Reply by Jairolaya12x** [reply]
Yes, so is possible to create the height mapping with bCNC via wifi? Or is it required to have the PC connected?

**Reply by Harvie** [reply, marked as answer]
bCNC supports remote connection. see: https://github.com/vlachoudis/bCNC/wiki/Connecting

**Reply by Jairolaya12x** [reply]
Thanks a lot
